Loop/Managers/DeviceDataManager.swift

```diff
@@ -1201,19 +1201,17 @@ extension DeviceDataManager {
     func handleRemoteNotification(_ notification: [String: AnyObject]) {
         if FeatureFlags.remoteOverridesEnabled {
             if let command = RemoteCommand(notification: notification, allowedPresets: loopManager.settings.overridePresets) {
-                loopManager.mutateSettings { settings in
-                    switch command {
-                    case .temporaryScheduleOverride(let override):
-                        log.default("Enacting remote temporary override: %{public}@", String(describing: override))
-                        settings.scheduleOverride = override
-                    case .cancelTemporaryOverride:
-                        log.default("Canceling temporary override from remote command")
-                        settings.scheduleOverride = nil
-                    }
+                switch command {
+                case .temporaryScheduleOverride(let override):
+                    log.default("Enacting remote temporary override: %{public}@", String(describing: override))
+                    loopManager.mutateSettings { settings in settings.scheduleOverride = override }
+                case .cancelTemporaryOverride:
+                    log.default("Canceling temporary override from remote command")
+                    loopManager.mutateSettings { settings in settings.scheduleOverride = nil }
                 }
-            } else {
-                log.info("Unhandled remote notification: %{public}@", String(describing: notification))
             }
+        } else {
```

Comment:
Did you mean to move this log to one block out? I think it belongs with the `if let command = RemoteCommand(...`.

Reply:
Or maybe my eyes are just having a hard time with figuring out which block it belongs to.